appium-uiautomator2-server/build.gradle

```diff
@@ -37,3 +37,17 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
 
 }
+
+task installAUT(type: Exec) {
+    def adb = android.getAdbExe().toString()
+    File configFile = file('../res/selendroid-test-app-debug.apk')
+    commandLine "$adb", 'install', '-r', "$configFile"
```

Comment:
`commandLine "$adb install -rg $apk".split(' ')`

I think it's cleaner to use split, we need the -g flag on api 23 and above, and config file isn't a good name for the apk variable.